--- /ext/gd/gd.c
+++ /ext/gd/gd.c
@@ -62,13 +62,13 @@

 static int le_gd, le_gd_font;

-#include <gd.h>
-#include <gd_errors.h>
-#include <gdfontt.h>  /* 1 Tiny font */
-#include <gdfonts.h>  /* 2 Small font */
-#include <gdfontmb.h> /* 3 Medium bold font */
-#include <gdfontl.h>  /* 4 Large font */
-#include <gdfontg.h>  /* 5 Giant font */
+#include "libgd/gd.h"
+#include "libgd/gd_errors.h"
+#include "libgd/gdfontt.h"  /* 1 Tiny font */
+#include "libgd/gdfonts.h"  /* 2 Small font */
+#include "libgd/gdfontmb.h" /* 3 Medium bold font */
+#include "libgd/gdfontl.h"  /* 4 Large font */
+#include "libgd/gdfontg.h"  /* 5 Giant font */

 #ifdef ENABLE_GD_TTF
 # ifdef HAVE_LIBFREETYPE
